src/persist-client/src/lib.rs

```diff
@@ -43,6 +43,7 @@ use crate::write::{WriteHandle, WriterId};
 pub mod batch;
 pub mod cache;
 pub mod error;
+mod paths;
```

Comment:
decided to move it under `impl` -- `ShardId` is already `pub` which seems like the only piece that an external caller might need access to. how we structure the keys under the hood feels sufficiently internal otherwise